pkg/providers/instancetype/types.go

```diff
@@ -118,10 +117,6 @@ func computeRequirements(ctx context.Context, info *ec2.InstanceTypeInfo, offeri
 	// GPU Labels
 	if info.GpuInfo != nil && len(info.GpuInfo.Gpus) == 1 {
 		gpu := info.GpuInfo.Gpus[0]
-		requirements.Get(v1alpha1.LabelInstanceAcceleratorName).Insert(lowerKabobCase(aws.StringValue(gpu.Name)))
-		requirements.Get(v1alpha1.LabelInstanceAcceleratorManufacturer).Insert(lowerKabobCase(aws.StringValue(gpu.Manufacturer)))
-		requirements.Get(v1alpha1.LabelInstanceAcceleratorCount).Insert(fmt.Sprint(aws.Int64Value(gpu.Count)))
-		requirements.Get(v1alpha1.LabelInstanceAcceleratorMemory).Insert(fmt.Sprint(aws.Int64Value(gpu.MemoryInfo.SizeInMiB)))
```

Comment:
Same here, don't we only need to remove this line?

Reply:
We're creating distinct groups of instance types that have accelerators and those that have gpus